File: 1. Labor/TypeScript/Rank.ts
class Rank {
  // Properties
  private rank_num: number; // adatelem
  private rank_nam: string; // adatelem

  public Rank_Number: number;
  public Rank_Name: string;

  constructor() {
    // Constructor
    this.Rank_Number = 0;
    this.Rank_Name = '';
  }

  // Property descriptions
  get Rank_Num(): number {
    return this.rank_num;
  }

  set Rank_Num(value: number) {
    this.rank_num = value;
  }

  get Rank_Nam(): string {
    return this.rank_nam;
  }

  set Rank_Nam(value: string) {
    this.rank_nam = value;
  }
}

File: 2. Labor/Python/Rank.py
class Rank:
    def __init__(self) -> None:
        # Empty constructor (can be omitted if not needed)
        pass

    # Properties
    @property
    def Rank_Number(self) -> int:
        return self.rank_num

    @Rank_Number.setter
    def Rank_Number(self, value: int) -> None:
        self.rank_num = value

    @property
    def Rank_Name(self) -> str:
        return self.rank_nam

    @Rank_Name.setter
    def Rank_Name(self, value: str) -> None:
        self.rank_nam = value
